docs/integrations/monitor-non-service-metrics-prometheus.rst

```diff
@@ -0,0 +1,101 @@
+Monitor non-service-specific metrics using Prometheus
+========================================================
+
+Prometheus integration allows you to monitor your Aiven services and understand the resource usage. Additionally, using this integration, you can monitor/track some non-service-specific metrics that may be worth monitoring. 
+
+To start monitoring these metrics, you need the :doc:`Prometheus integration configured and set up the Prometheus server <../platform/howto/integrations/prometheus-metrics>`.
+
+The complete list of metric you can monitor are not listed here. However, you can get the complete list in the Prometheus service or by sending ``HTTP GET`` with ``cURL`` as shown below: 
+
+.. code:: 
+  
+    curl --insecure --user promXXXX:XXXXXXXXXXXXXXXX https://YOUR-SERVICE-FQDN:9273/metrics
+
```

Comment:
Please add an explanation of what this command outputs and how the user can use the information returned.

Reply:
I will include this; I need to get this information as I don't have the environment setup for this.